### DISCUSSION TITLE
Calliope smart playlist support

### DISCUSSION THREAD

**Discussion by Rudd-O** [music-assistant/support#2287] [Feature requests and ideas]
https://github.com/ZeroByteOrg/calliope is an amazing playlist generator that supports multiple music sources just like Music Assistant does.

It would be simply phenomenal if Calliope support could be added as a provider for Music Assistant.

**Comment by sirbooker** [comment]
i would love to see a playlist generator which could say  generate by genre or generate 60mins,120 mins, something like this would be awesome
